dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,148 @@
+
+</blockquote>
+
+![A screenshot of a GitHub pull request title "Version Packages"](./images/lit-release-process/version-packages-pr.png)
+
+4. Request code reviews from members of the team familiar with the packages that will be released.
+
+   1. The purpose of this pull request is to:
+      1. Delete all the changeset files, moving their contents into the various Changelog files.
+      1. Bump the `window.litElementVersions` global.
+      1. Bump version numbers in package.json files.
+      1. Bump dependency ranges in related package.json files if needed. (Only expected in minor or major releases).
+   1. Errors to look for:
+      1. When a new package is created and released, it is easy to accidentally have the version in the package.json set to vesion 1.0.0, resulting in Changeset bumping the version to 2.0.0. Fix is to create the package at version 0.0.0.
+      1. Major version releases or minor features may require further scrutiny as major releases mark major events and require coordination.
+      1. Changelog updates to `lit` core libraries such as `lit-html` should be relected in the umbrella `lit` package.
+      1. Internal private or not ready to be released packages should not get changelogs or version bumps.
+   1. Fix errors by closing the automated "Version Packages" pull request. Commit changes to `main` branch and re-run the release process.
+
+5. Once "Version packages" is approved, (provided no new changesets were merged to `main` in the meantime), squash and merge.
+
+<blockquote>
+
+⚠️ If a commit to `main` occurred while the "Version Packages" PR was open and a new changeset was created then do not squash and merge.
+Instead re-run version packages to update the PR with the most recent changes. Otherwise the following steps will not publish a release and will instead create another "Version Packages" PR integrating the new changeset file.
+
+A publish only occurs when there are no unreleased changeset files on `main`.
+
+</blockquote>
+
+6. The process to publish to NPM and create a GitHub release follows the same process as before. Find the "Version Packages" commit and wait for at least `tests-local` to pass. Then repeat steps 1. and 2. from above, where this time the result of approving the "Release / Release" GitHub Action is publishing a new NPM version release and GitHub release.
+
+![GitHub commits image showing pending GitHub actions on Version Packages commit](./images/lit-release-process/version-packages-commit-actions.png)
+
+![GitHub Release / Release action page to deploy the Version Packages commit](./images/lit-release-process/version-packages-release-action-ui.png)
+
+![Github publish Lit release modal with "Approve and deploy" button](./images/lit-release-process/publish-lit-release-modal.png)
+
+7. Check https://github.com/lit/lit for the new release notes. Congratulations you've released Lit.
```

Comment:
By "release notes" I think we mean entries https://github.com/lit/lit/releases right? Maybe we can just do that link. Also, checking https://www.npmjs.com/ for confirming actual publish to npm would be a good callout.

Reply:
Excellent suggestion!